airflow/providers/dbt/cloud/operators/dbt.py

```diff
@@ -128,6 +130,7 @@ def execute(self, context: Context):
             self.trigger_reason = (
                 f"Triggered via Apache Airflow by task {self.task_id!r} in the {self.dag.dag_id} DAG."
             )
+        self._validate_trigger_reason()
```

Comment:
Its does trimming the reason along with validation. Better to rename it to reflect the same.